packages/data-models/appConfig.ts

```diff
@@ -83,13 +83,15 @@ const APP_ROUTE_DEFAULTS = {
   ],
 };
 
-export type IHeaderColourOptions = "primary" | "secondary" | "none";
+export type IHeaderFooterBackgroundOptions = "primary" | "secondary" | "none";
 export type IHeaderVariantOptions = "default" | "compact";
 
 const APP_HEADER_DEFAULTS = {
-  title: "App",
   collapse: false,
-  colour: "primary" as IHeaderColourOptions,
+  colour: "primary" as IHeaderFooterBackgroundOptions,
+  show: true,
+  template: null as string | null,
```

Comment:
nit(non-blocking): I'm not quite sure what casting the `null` to string really achieves here. If it's an optional property could just leave as `undefined`, unless `null` specifically differentiates from `undefined` in another meaningful way I'm missing here

Reply:
Oh, actually I see it's because we're defining as a const and then deriving types from it. Now that the list of defaults is getting quite long and specific I think it would be better to first define the header config type definition and then assigning the defaults from the type (save all the additional type-casting)